README.md

```diff
@@ -63,48 +70,40 @@ let loader = require('taskcluster-lib-loader');
 
 // Create loader
 let load = loader({
-  // Definition of 'server' component
-  server: {
-    required: ['port'],
-    setup: async (ctx) => {
-      let server = http.createServer();
-      await new Promise((accept, reject) => {
-        server.once('error', reject);
-        server.once('listening', accept);
-        server.listen(ctx.port);
-      });
-      return server;
-    }
-  },
-  
-  // Definition of 'express' component, notice that even through the 'server'
-  // components setup function returns a promise, `ctx.server` is a server
-  // object as resolved.
-  express: {
-    required: ['server'],
-    setup: (ctx) => {
-      let app = express();
-      // setup routes...
-      ctx.server.on('request', app);
-      return app;
-    }
-  }
+  server: { /* ... */ },
+  express: { /* ... */ },
 
-  // Virtual components must always be specified for `load` to work
+  // Virtual components that must always be specified for `load` to work
 }, ['port']);
 
 // Create express (here we're forced to specify port)
 let server = await load('express', {port: 80});
 ```
 
-As a neat little the load has a default target `graphviz` which returns a
-graphical representation of the dependency graph in graphviz format.
+As a neat little treat, the load has a default target `graphviz` which returns
+a graphical representation of the dependency graph in graphviz format.
```

Comment:
this is not being changed in this review, but while you're editing the file, we should make the wording more correct.  Something like "...which returns a representation of the dependency graph in graphviz's dot format.  This representation can be rendered using the graphviz tool"